### ISSUE TITLE
Ability to change default route for API/Content etc

### ISSUE THREAD

**Issue by ccelson** [dropbox/dropbox-sdk-dotnet#24]
I'm needing to hit the dropbox API's through a jump server to get around some security. It would be nice to be able to override the default hostname for the different domains. Is that something that could be done, or are there security implications that I'm not seeing

**Comment by greg-db** [comment]
I don't believe the SDK currently offers a way to do that, but we'll consider this a feature request. 

The SDK does do certificate pinning though, which would probably need to be bypassed to enable this.

**Comment by ccelson** [comment]
Thanks, I was going to fork the repo just for a temporary work around. You think we'd run into troubles with the cert pinning?

**Comment by greg-db** [comment]
Yes, if you connected to a different server when the Dropbox certificates were expected, certificate pinning would fail. If you're forking it anyway though, you could certainly disable that, though it's not recommended.
